resources/ditaotdaykeys/dita-ot-day-2019.ditamap

```diff
@@ -0,0 +1,111 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
+<map>
+  <!-- This file was generated from dita-ot-day-ditamap-keys.py. -->
+  <title>DITA-OT Day 2019</title>
+  <keydef keys="DITA-OTShopTalk-2019" href="https://www.oxygenxml.com/events/2019/dita-ot_day.html#easy_as_pie_pdf_customizations_using_css" format="html" scope="external">
```

Comment:
Looks like the link target IDs in this file are each off by one. 

This one points to the following presentation, and each of the others as well. The keys reflect the presentation titles, but the links each point to the next presentation in the list, instead of to the intended targets.

Reply:
This is resolved with [f3f1f7b](https://github.com/dita-ot/docs/pull/304/commits/f3f1f7b866dc71bb8f2eac889e517735c9805af3)...after I first attempted with [877cc9b](https://github.com/dita-ot/docs/pull/304/commits/877cc9b97f495f1202d4f3f3f40b80ba6582defd).

The issue is a lack of consistency in the HTML. The Welcome session does not have an agenda-author or summary description. This threw the count off by one for the href, because there is a video and an accompanying link.